### PR TITLE
shell_commands: add command for packet buffer stats

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
I'm using something like this command for a while now for debugging
GNRC. Usually, I just patch it into the application I'm using it with,
but I think there is a benefit to also provide it to RIOT upstream
properly.

### Issues/PRs references
None

### CODE REVIEW

File: sys/shell/commands/sc_gnrc_pktbuf.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+#include "net/gnrc/pktbuf.h"
+
+int _gnrc_pktbuf_cmd(int argc, char **argv)
+{
+    (void)argc;
+    (void)argv;
+    gnrc_pktbuf_stats();
+}
```
Comment (cgundogan) [inline sys/shell/commands/sc_gnrc_pktbuf.c]:
```c
return 0
```

Comment (miri64) [reply sys/shell/commands/sc_gnrc_pktbuf.c]: Done

Comment (miri64) [reply sys/shell/commands/sc_gnrc_pktbuf.c]: Wondering why it was compiling though :-/

**Comment by miri64** [comment]
Squashed

**Pull request merged**